### ISSUE TITLE
logger info is same every two lines

### ISSUE THREAD

**Issue by greathope** [tensorflow/models#4683]
very weird, I did not change anything in `train.py`, anyone?

>INFO:tensorflow:Restoring parameters from training_frcnn/model.ckpt-500000
INFO:tensorflow:Restoring parameters from training_frcnn/model.ckpt-500000
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Starting Session.
INFO:tensorflow:Starting Session.
INFO:tensorflow:Saving checkpoint to path training_frcnn/model.ckpt
INFO:tensorflow:Saving checkpoint to path training_frcnn/model.ckpt
INFO:tensorflow:Starting Queues.
INFO:tensorflow:Starting Queues.
INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:Recording summary at step 500000.
INFO:tensorflow:Recording summary at step 500000.
INFO:tensorflow:global step 500001: loss = 0.1333 (7.742 sec/step)
INFO:tensorflow:global step 500001: loss = 0.1333 (7.742 sec/step)
INFO:tensorflow:global step 500002: loss = 0.1147 (0.105 sec/step)
INFO:tensorflow:global step 500002: loss = 0.1147 (0.105 sec/step)
